Makefile

```diff
@@ -98,6 +123,36 @@ docker-build-logger: test
 docker-push-logger:
 	docker push ${LOGGER_IMG}
 
+docker-build-sklearn: 
+	cd python && docker build -t ${KO_DOCKER_REPO}/${SKLEARN_IMG} -f sklearn.Dockerfile .
```

Comment:
I think we need to use `pushd/popd` instead of `cd` here

Reply:
Thanks for advice. In fact makefile is designed to work with cd well in which cd only take effect in the current line, so no need to popd or cd back. In fact pushd does not work well in Makefile and you must use it in one line with popd too... Can refer
https://stackoverflow.com/questions/32302228/pushd-not-working-in-makefile